README.adoc

```diff
@@ -10,3 +10,56 @@ image:http://www.javadoc.io/badge/{groupId}/{artifactId}.svg["Javadocs", link="h
 A Java project developed during the https://github.com/oliviercailloux/java-course[Java course] taught at Université Paris-Dauphine (in http://www.mido.dauphine.fr/[MIDO] M2 MIAGE).
 
 The code is hereby published under the MIT License, with their permission.
+
+
+== PLAN
+After the 4 iterations, we would like to improve the existing code as much as possible (optimization, new features).
```

Comment:
Why “after”? Do you mean “during”?

Reply:
@oliviercailloux Yes "during" is the word.